qiskit/aqua/operators/gradients/hessian_base.py

```diff
@@ -0,0 +1,67 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The module to compute Hessians."""
+
+from typing import Union
+
+from qiskit.aqua.operators.gradients.circuit_gradients.circuit_gradient \
+    import CircuitGradient
+from qiskit.aqua.operators.gradients.derivative_base import DerivativeBase
+
+
+class HessianBase(DerivativeBase):  # pylint: disable=abstract-method
+    """Base class for the Hessian of an expected value."""
+
+    def __init__(self,
+                 hess_method: Union[str, CircuitGradient] = 'param_shift',
+                 **kwargs):
+        r"""
+        Args:
+            hess_method: The method used to compute the state/probability gradient. Can be either
+                ``'param_shift'`` or ``'lin_comb'`` or ``'fin_diff'``.
+                Deprecated for observable gradient.
```

Comment:
Similar comment around this deprecated text as noted in GradientBase

Reply:
see above